README.md

```diff
@@ -57,75 +97,11 @@ Here's a quick taste of the technologies a standard Redwood application will
 use:
 
 - [React](https://reactjs.org/)
-- [GraphQL](https://graphql.org/) ([GraphQL Yoga](https://www.graphql-yoga.com) + [Envelop](https://www.envelop.dev) + [Apollo Client](https://www.apollographql.com/docs/react))
 - [Prisma](https://www.prisma.io/)
 - [Jest](https://jestjs.io/)
 - [Storybook](https://storybook.js.org/)
-- [Babel](https://babeljs.io/)
-- [Webpack](https://webpack.js.org/)
-- [Fastify](https://www.fastify.io)
 - [Pino](https://getpino.io)
 
```

Comment:
I'd be fine to remove this section because it feels limiting. If we keep it, let's add these at a minimum:
- Vite: https://vitejs.dev
- GraphQL: Yoga (Server) and Apollo (Client), https://the-guild.dev/graphql/yoga-server and https://www.apollographql.com/docs/react/

Reply:
Done.